src/GitHub.App/Api/ApiClient.cs

```diff
@@ -84,7 +84,7 @@ public IObservable<UserAndScopes> GetUser()
             }
             else
             {
-                log.Error($"Error reading scopes: /user succeeded but {ScopesHeader} was not present.");
+                log.Error("Error reading scopes: /user succeeded but {ScopesHeader} was not present.", ScopesHeader);
```

Comment:
This changes the log message from string interpolation to a normal params method, but `{ScopesHeader}` is still in the string so this is probably not going to work. Is there a particular reason this change is needed?